west.yml

```diff
@@ -172,6 +174,10 @@ manifest:
       revision: 07c8d591b47c4c3609f29871be031c9af39401e1
       groups:
       - find-my
+    - name: memfault-firmware-sdk
+      path: modules/memfault-firmware-sdk
```

Comment:
Should this not be under: `modules/lib/memfault-firmware-sdk`

How is this repo different from, for example:
https://github.com/nrfconnect/sdk-nrf/blob/69059c35477cc7fd431880e0d7f744f9fa509e71/west.yml#L161
https://github.com/nrfconnect/sdk-nrf/blob/69059c35477cc7fd431880e0d7f744f9fa509e71/west.yml#L167
https://github.com/nrfconnect/sdk-nrf/blob/69059c35477cc7fd431880e0d7f744f9fa509e71/west.yml#L153
https://github.com/nrfconnect/sdk-nrf/blob/69059c35477cc7fd431880e0d7f744f9fa509e71/west.yml#L136

Reply:
as memfault is already a Zephyr module, I think the nrf/module/memfault location is fine.